This is a 0x05. Python - Exceptions task
